docs/envvars.rst

```diff
@@ -74,6 +74,14 @@ applicable.
       - xonsh.environ.FORMATTER_DICT  
       - Dictionary containing variables to be used when formatting PROMPT and TITLE 
         see `Customizing the Prompt <tutorial.html#customizing-the-prompt>`_.
+    * - HISTCONTROL
+      - ``''``
```

Comment:
I think that this default value should be updated to be an empty set, rather than an empty string.